plugins/woocommerce/client/blocks/assets/js/base/stores/woocommerce/cart.ts

```
@@ -22,680 +22,815 @@ import type {
+        collectiveTimer = setTimeout(() => {
+            actions.processCollectiveActions();
+        }, delay);
+    }
 };
 
 let pendingRefresh = false;
 let refreshTimeout = 3000;
 
+// --- GLOBAL REQUEST COUNTER ---
+// Used to prevent race conditions by ignoring outdated server responses.
+let _requestCounter = 0;
```

Biome (2.3.13)

[error] Shouldn't redeclare '_requestCounter'. Consider to delete it or rename it.

'_requestCounter' is defined here:


(lint/suspicious/noRedeclare)